build.go

```diff
@@ -71,7 +73,8 @@ func newBuildCommand() *cobra.Command {
 	fs.BoolVar(&build.noConsole, "no-console", false, "Use non-console progress UI")
 	fs.BoolVar(&build.noCache, "no-cache", false, "Do not use cache when building the image")
 	fs.StringVarP(&build.output, "output", "o", "", "BuildKit output specification (e.g. type=tar,dest=build.tar)")
-	fs.Var(build.cacheFrom, "cache-from", "Images to consider as cache sources")
+	fs.Var(build.cacheFrom, "cache-from", "Buildkit import-cache or Buildx cache-from specification")
+	fs.Var(build.cacheTo, "cache-to", "Buildkit export-cache specification")
```

Comment:
export-cache -> cache-to

Reply:
This is what you mean?

```suggestion
	fs.Var(build.cacheTo, "cache-to", "Buildkit export-cache -> cache-to specification")
```

Reply:
No, should be "Buildx cache-to specification"